### ISSUE TITLE
net.fortuna.ical4j.util.Configurator doesn't close the InputStream for "ical4j.properties"

### ISSUE THREAD

**Issue by symVmueller** [ical4j/ical4j#248]
**Description**
net.fortuna.ical4j.util.Configurator retrieves an InputStream in its static initialization code block, that never is closed. This leads to Warnings when undeploying an application that uses ical4j in glassfish.

**Expected behavior**
No Warnings. The InputStream is closed after using it.

**Environment:**
 - OS: Ubuntu 16.04
 - Java version: 8_u161
 - Application server: Payara 4.174
 - ical4j version: 3.0.0

**Additional context**
Partial Stack Trace:
```
[javax.enterprise.system.util] [tid: _ThreadID=98 _ThreadName=AutoDeployer] [timeMillis: 1529675310651] [levelValue: 900] [[
  Input stream has been finalized or forced closed without being explicitly closed; stream instantiation reported in following stack trace
java.lang.Throwable
        at com.sun.enterprise.loader.ASURLClassLoader$SentinelInputStream.<init>(ASURLClassLoader.java:1284)
        at com.sun.enterprise.loader.ASURLClassLoader.getResourceAsStream(ASURLClassLoader.java:942)
        at net.fortuna.ical4j.util.ResourceLoader.getResourceAsStream(ResourceLoader.java:79)
        at net.fortuna.ical4j.util.Configurator.<clinit>(Configurator.java:58)
        at net.fortuna.ical4j.util.CompatibilityHints.<clinit>(CompatibilityHints.java:87)
        at net.fortuna.ical4j.model.Iso8601.<init>(Iso8601.java:78)
        at net.fortuna.ical4j.model.Date.<init>(Date.java:150)
        at net.fortuna.ical4j.model.DateTime.<init>(DateTime.java:262)
...
```

**Comment by benfortuna** [comment]
Fix applied in PR #251

**Comment by benfortuna** [comment]
Fix applied. Closing issue.
